doc/guides/getting-started/flashing.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Flashing a RIOT Application
+description: Flash a RIOT application to a real hardware device
+---
+
+import Contact from '@components/contact.astro';
+
+<Contact />
+
+After all of the previous sections are completed, we can finally flash some
+real hardware. In this case, we use an `esp32-mh-et-live-minikit` development
+board. The guide should mostly apply to all other boards as well.
+
+:::note
+    Some boards require extra steps to be flashed, such as pressing a button
+    to enter a bootloader or attaching an external programmer. Refer to the
+    documentation of the board to check if extra steps are required.
+:::
+
+This assumes that the USB UART bridge of the ESP32 development board has
+been attached to WSL and VS Code has been launched from within WSL by running
+`code .` inside the RIOT repository from the Ubuntu terminal.
```

Comment:
I do not understand.

Reply:
Oh is this entire guide windows specific? We should reflect that in the filename and introduction of this guide.

Reply:
Thats an oversight on my part, the guide, for the most part, is platform-agnostic. I will change that :+1: